### PR TITLE
Upgrade xtt to v0.10.0

### PR DESCRIPTION
**Pull request by zanebeckwith**

### CODE REVIEW

File: _build_xtt.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
 
 # Note: Reset POST to 0 on version bump
-XTT_VERSION = 'v0.9.2'
-POST        = '2'
+XTT_VERSION = 'v0.10.0'
+POST        = '1'
```
Comment (drbild) [inline _build_xtt.py]: Reset `POST` to 0, not 1.

**Review by drbild — CHANGES_REQUESTED** [review]
Just the wrong `POST`, otherwise LGTM.

File: _build_xtt.py (hunk 1 above)
Comment (zanebeckwith) [reply _build_xtt.py]: Does this not track the deb version?

Comment (drbild) [reply _build_xtt.py]: No, it tracks the XTT version. 

The POST here is for the Python packaging revision.  The "REV" in a Debian package is for the Debian packaging revision. Those move independently  (depending on how many mistakes I make while packaging for a particular platform!).

(Debian revisions appear to be 1-indexed, while Python packaging revisions are 0-indexed )

Comment (zanebeckwith) [reply _build_xtt.py]: Gotcha

Done

**Review by drbild — APPROVED** [review]
LGTM.

**Comment by zanebeckwith** [comment]
Also fixed an issue with our travis linux build, to allow changes to the patch-number in the python-3.6 directory.

**Pull request merged**